### ISSUE TITLE
Delete this repo

### ISSUE THREAD

**Issue by mattwynne** [cucumber-ltd/heroku-buildpack-subversion#2]
@aslakhellesoy we don't need this anymore do we?

**Comment by aslakhellesoy** [comment]
We don't, no. @paul-hammant want to take it over?

**Comment by paul-hammant** [comment]
Not really. While I'm interested in WebDav on Subversion, I'm not skilled enough to maintain a Docker-esque thing like that.
